installations/InstallationsExample/ViewController.m

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "ViewController.h"
+
+// TODO: Remove once FirebaseInstallations released.
```

Comment:
Will this TODO be resolved in time for the merge?

Reply:
@morganchen12  I would prefer to merge the PR with the TODOs and update the code right after M61. WDYT?

Reply:
SGTM